src/three/plugins/SUBTREELoader.js

```diff
@@ -0,0 +1,747 @@
+/**
+ * Structure almost identical to Cesium, also the comments and the names are kept
+ * https://github.com/CesiumGS/cesium
```

Comment:
Is there a specific file this is from? How much of this code is copy-pasted? I'd recommend taking a look at the Cesium license to make sure we're following what's expected under their Apache-2 license:

https://github.com/CesiumGS/cesium?tab=Apache-2.0-1-ov-file#readme

Reply:
https://github.com/CesiumGS/cesium/blob/main/packages/engine/Source/Scene/Implicit3DTileContent.js

The structure is the same. Parsing/reading the json/binary -> Create children -> Filling the uri if a content/subtree file is available. I've kept the names and comment for clarity.
The license is the same as this repo, which is very permissive.

Reply:
Thanks - I still need to read through the SUBTREELoader to make sure I understand everything and am reading through the spec to make sure everything lines up, as well. But we should add a link using the latest commit hash to this file in the comment so it's clear what was referenced.

Thanks for bearing with me on the questions and changes. This is a pretty complex addition.

Reply:
I'll add the permalink
https://github.com/CesiumGS/cesium/blob/0a69f67b393ba194eefb7254600811c4b712ddc0/packages/engine/Source/Scene/Implicit3DTileContent.js

> Thanks for bearing with me on the questions and changes. This is a pretty complex addition.

It's my first contribution to your lib, I'm not used to its code, so I was expecting this back and forth :+1: